openworld/sdk/generator/client/model.py

```diff
@@ -31,6 +31,14 @@ class Argument(fastapi_code_generator_parser.Argument):
     in_: ParamTypes = None
     alias: str = ""
     description: str = ""
+    datatype: typing.Optional[DataType] = None
+
+    # Replace `cached property` with a normal `property` in order to take effect on change.
```

Comment:
What's a `cached property`?

Reply:
`[Cached Property](https://www.educative.io/answers/what-is-the-cachedproperty-decorator-module-in-python)` is a decorator equivalent to `@property`, but it's value is cached.